MediaBrowser.Controller/ClientEvent/ClientEventLogger.cs

```diff
@@ -23,8 +23,12 @@ public async Task<string> WriteDocumentAsync(string clientName, string clientVer
         {
             var fileName = $"upload_{clientName}_{clientVersion}_{DateTime.UtcNow:yyyyMMddHHmmss}_{Guid.NewGuid():N}.log";
             var logFilePath = Path.Combine(_applicationPaths.LogDirectoryPath, fileName);
-            await using var fileStream = new FileStream(logFilePath, FileMode.CreateNew, FileAccess.Write, FileShare.None);
-            await fileContents.CopyToAsync(fileStream).ConfigureAwait(false);
+            FileStream fileStream;
+            await using (fileStream = new FileStream(logFilePath, FileMode.CreateNew, FileAccess.Write, FileShare.None))
```

Comment:
`await using (var fileStream = new FileStream(logFilePath, FileMode.CreateNew, FileAccess.Write, FileShare.None))` would be better... no reason for distinct declaration.

Reply:
I was not able to resolve the warning without writing it this way.